isc24_tutorial/isc24_tutorial_proposal.md

```diff
@@ -0,0 +1,164 @@
+- Be familiar with a technical approach to collaboratively developing open source lessons based on a template.
+
+
+### Detailed description of the tutorial content (Maximum pages: 3)
+
+**Overview and goals of the tutorial (takeaways for the audience)**
+
+This tutorial will cover best practices in developing training material and
+delivering technical training workshops. Participants will get acquainted with
+key concepts from pedagogical science and the philosophy of teaching, learn
+practical methods for teaching interactive and engaging hands-on workshops
+which maximise learning through careful consideration of cognitive load,
+acquire skill in developing effective training lessons that facilitate memory
+consolidation, and get practical tips and tricks on how to effectively organise
+in-person or online workshops.
+
+After attending this tutorial, participants will:
+
+- Have increased confidence in teaching interactive workshops in online and in-person settings.
+- Be familiar with important pedagogical concepts and scientific findings about how people learn new skills.
+- Appreciate the importance of frequent hands-on practice in building skill.
+- Understand how learning can be improved by following certain best practices in interactive teaching.
+- Know how to backwards-design training material based on intended learning outcomes.
+- Know about important differences between online and in-person training and how inherent challenges in online training can be overcome.
+- Be familiar with a technical approach to collaboratively developing open source lessons based on a template. 
+
+
+**Targeted audience**
+
+Teaching is a profession, but also something that everyone needs to be able to
+do to some degree, since everyone has their own personal specialties they will
+either teach or mentor.
+
+This course can be relevant for different learner personas:
+
+- **You run a practical teaching program** at your institution (for example as part of a research computing group) and would like to learn best practices for collaborative teaching of technical topics, so that you aren’t re-inventing the same thing over and over again.
+- **You are a technical specialist** who is frustrated with the way you currently try to teach others who need to use your software or infrastructure. You can’t spend too much time to become a professional, but you know you need something more than what you’ve been doing. Thus, you would like to adopt some of the best practices of designing and teaching interactive, hands-on workshops.
+- **You’ve been teaching alone**, but would like to join a collaboration network for more co-teaching and to reduce the amount of duplication of effort.
+- **You are interested in teaching CodeRefinery/ENCCS/... lessons**, and would like a comprehensive kick-start to how these workshops work, either to join and teach with us, or teach our lessons independently.
+
+**Detailed outline of the tutorial (with time slots)**
```

Comment:
Are the breaks pre-set by the conference? Or some of them? I would otherwise suggest to re-balance a bit but maybe there is no wiggle-room.

Reply:
yes the 30 minute breaks are pre-set by the conference. I prioritised having the breaks after full episodes rather than splitting episodes across a break, but the result is that the 10 minute breaks are not well separated from the 30 minute breaks. I'll see if it can be improved, but maybe this isn't critical and during the workshop we can improvise the shorter breaks